### PR TITLE
Feature/574 add hotkeys to dialogs

### PR DESCRIPTION
**Pull request by sebastianconcept**
Pairing with @jpuri on this card
https://trello.com/c/tFpuBZ7d/574-add-hotkeys-to-dialogs

### CODE REVIEW

File: client/source/js/modules/ui/modal/modal-service.js [hunk 1]
```diff
@@ -21,15 +21,24 @@ define([
          * executes the right callback depending on the user's choice.
          */
         confirm: function (onAccepted, onRejected, acceptButton, rejectButton, message, title) {
-          $modal.open({
+
+          var onModalKeyDown = function (anEvent) {
```
Comment (nikgraf) [inline client/source/js/modules/ui/modal/modal-service.js]: @jpuri @sebastianconcept Why `anEvent` instead of just `event`?

Comment (nikgraf) [inline client/source/js/modules/ui/modal/modal-service.js]: Just found this: http://stackoverflow.com/a/1510106/837709

Comment (sebastianconcept) [inline client/source/js/modules/ui/modal/modal-service.js]: The convention of using `a` or `an` or `someBlah` is explicitely referring to an instance(s). When used consistently, it makes argument naming less ambiguous.

**Comment by nikgraf** [comment]
Works like a charm! thx @sebastianconcept @jpuri

File: client/source/js/modules/ui/modal/modal-service.js (hunk 1 above)
Comment (sebastianconcept) [inline client/source/js/modules/ui/modal/modal-service.js]: And I forgot to mention that, that naming together with good method naming, makes it closer to spoken english:
`this.doSomethingWith(anObject)`
`this.convertUsing(aConverter)`
`moneyController.includes(somePayments)`
`carModel.removeTires(someTires)`
etc

**Pull request merged**